npm/private/npm_translate_lock_state.bzl

```diff
@@ -150,7 +150,7 @@ def _init_pnpm_labels(label_store, rctx):
     #
     # TODO: Try to understand this better and see if we can go back to using
```

Comment:
since we're here and studying this, could we remove the TODO as well?

Reply:
I think that's out of scope here and I'd rather not put the time in atm.